palisade-service/src/main/java/uk/gov/gchq/palisade/service/palisade/service/SimplePalisadeService.java

```diff
@@ -81,20 +82,41 @@ public SimplePalisadeService(final AuditService auditService, final UserService
         final RequestId originalRequestId = request.getId();
         LOGGER.debug("Registering data request: {}, {}", request, originalRequestId);
 
+        AuditRequest.RegisterRequestExceptionAuditRequest.IException auditException = AuditRequest.RegisterRequestExceptionAuditRequest
+                .create(originalRequestId)
+                .withUserId(request.getUserId())
+                .withResourceId(request.getResourceId())
+                .withContext(request.getContext());
```

Comment:
I don't like that this object will be created for every registerDataRequest but only gets used if an exception occurs which should be a rare occurrence

Reply:
I'll refactor this a little